pyk/src/pyk/ktool/kfuzz.py

```diff
@@ -20,13 +21,34 @@
     from ..kore.syntax import Pattern
 
 
+class KFuzzTestCaseHandler(ABC):
+    @abstractmethod
+    def handle_test_case(self, args: Mapping[EVar, Pattern]) -> None: ...
+
+    @abstractmethod
+    def handle_failure_case(self, args: Mapping[EVar, Pattern]) -> None: ...
+
+
+class _KFuzzDefaultTestCaseHandler(KFuzzTestCaseHandler):
```

Comment:
What do you think about `NullTestCaseHandler` (as a subclass of `KFuzzTestCaseHandler` or `TestCaseHandler`) or `KFuzzNullHandler` (as a subclass of `KFuzzHandler`).